### ISSUE TITLE
False positive for captioned <video> without <track>

### ISSUE THREAD

**Issue by jasonkiss** [dequelabs/axe-core#285]
The "<video> elements must have a <track> for captions" rule appears to equate lack of a `<track>` element with lack of captioning. But captions can be embedded in the video, or provided otherwise (e.g. YouTube captions) without the need for a `<track>` element within the `<video>`.

**Comment by marcysutton** [comment]
Thanks for your issue @jasonkiss. It does look like Youtube captions are added in a random DIV element, a la:
 ```
<div class="caption-window ytp-caption-window-bottom ytp-caption-window-rollup"
id="caption-window-1" dir="ltr" tabindex="0" aria-live="assertive" lang="en" 
draggable="true" data-layer="5" 
style="touch-action: none; text-align: left; left: 60.8369%; width: 508px; height: 38px; margin-left: -254px; bottom: 12.207%;">
<span class="captions-text"></span></div>
```

The `<track>` element would be a more standards-focused application, but IMO captions used in this manner or embedded in a video are just as usable by deaf or hard-of-hearing people. Perhaps we should move this to a Best Practice rule instead @dylanb @WilcoFiers? Sine there aren't reliable ways to check for custom caption DIVs or captions embedded in videos.

**Comment by jasonkiss** [comment]
WFM. 

I wonder if there's a similar issue with audio description and the aXe rule for that?

**Comment by dylanb** [comment]
Our plan is to move these rules to the "needs review" category once that lands in the aXe extensions

**Comment by marcysutton** [comment]
Oh yes. That will be perfect!

**Comment by JaredNoble** [comment]
I can confirm that the audio element also generates the same result for a missing track tag.

**Comment by marcysutton** [comment]
This has been fixed internally, and will be in our next release.

**Comment by marcysutton** [comment]
Fixed with https://github.com/dequelabs/axe-core/commit/30be884ab9af790c8c80194906a387274d336d7c . Stay tuned for a release of axe-core 2.2.0
